parsons/zoom/zoom.py

```diff
@@ -602,7 +606,11 @@ def _get_request(self, endpoint: str, data_key: str, params: dict = {}, **kwargs
             if next_page_token:
                 params["next_page_token"] = next_page_token
 
-            r = self.client.get_request(endpoint, params=params, **kwargs)
+            try:
+                r = self.client.get_request(endpoint, params=params, **kwargs)
+            except InvalidClientError:
```

Comment:
Could this cause a recursive loop if the issue is something different than just needing to reauthenticate?

Reply:
Oh maybe not, just realized these are two separate functions - so this is just going to give you a single attempt to re-authenticate